config.go

```diff
@@ -118,8 +150,36 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Multiple networks can't be selected simultaneously.  Count number of
+	// network flags passed and assign active network params.
+	numNets := 0
+	if cfg.MainNet {
+		numNets++
+		activeNetParams = &decredMainNetParams
+	}
+	if cfg.TestNet {
+		numNets++
+		activeNetParams = &decredTestNet3Params
+	}
+	if cfg.SimNet {
+		numNets++
+		activeNetParams = &decredSimNetParams
+	}
+	if numNets > 1 {
+		str := "%s: mainnet, testnet and simnet params can't be " +
+			"used together -- choose one of the three"
+		err := fmt.Errorf(str, funcName)
+		fmt.Fprintln(os.Stderr, err)
+		fmt.Fprintln(os.Stderr, usageMessage)
+		return nil, nil, err
+	}
+
+	// Update LndNode using the activeNetParams
+	cfg.LndNode = "localhost:" + activeNetParams.rpcPort
```

Comment:
This is overriding `cfg.LndNode`, so if I run with `--lnd_node 10.20.30.40:10000` the faucet won't be able to connect to the lnd node.